cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServletUtils.java

```diff
@@ -55,10 +57,29 @@ static String getProposalSummary(GoalOptimizer.OptimizerResult result) {
       }
     }
     return String.format("%n%nThe optimization proposal has %d replica(%d MB) movements and %d leadership movements "
-                             + "based on the cluster model with %d recent snapshot windows and %.3f%% of the partitions "
-                             + "covered.", numReplicaMovements, dataToMove, numLeaderMovements,
-                         result.clusterModelStats().numSnapshotWindows(),
-                         result.clusterModelStats().monitoredPartitionsPercentage() * 100);
+            + "based on the cluster model with %d recent snapshot windows and %.3f%% of the partitions " + "covered.",
+            numReplicaMovements, dataToMove, numLeaderMovements, result.clusterModelStats().numSnapshotWindows(),
+            result.clusterModelStats().monitoredPartitionsPercentage() * 100);
   }
 
+  static Map<String, Object> getProposalSummaryInJson(GoalOptimizer.OptimizerResult result) {
```

Comment:
Would the following change make our intention regarding the use of the returned result more clear? `getProposalSummaryInJson` -> `getProposalSummaryForJson`